### DISCUSSION TITLE
Unable to Display Metadata Columns in PnP Search Result Web Part in SharePoint i.e. Date Time Column, Managed Metadata Column

### DISCUSSION THREAD

**Discussion by santoshk10** [microsoft-search/pnp-modern-search#3861] [Q&A]
Hi All,

I created a SharePoint Document Library that contains several columns, including Single Line, Date Time, and Managed Metadata.
![image](https://github.com/user-attachments/assets/d50f8e62-080a-4792-a9e8-4ec41f492a03)
![image](https://github.com/user-attachments/assets/566ef17f-0a7e-4853-b584-8ea1a7fe8508)

I want to display some of this metadata in the PnP Search Result web part in List View type.
![image](https://github.com/user-attachments/assets/df57c6f1-1481-497d-8991-22a2b6c4423c)

For example, I want to show:
**Below File Name**
**Article source : <Managed Metadata Column>
Year: <Date Time Column>**

We have already mapped all columns using Crawled Properties.
![image](https://github.com/user-attachments/assets/21d10df8-7596-422d-8e50-28a873aa55cf)
However, we are unable to get these columns in the Search Result.

Kindly guide us as we have been stuck on this for a long time.

Thank you!

**Comment by kasperbolarsen** [comment]
A precondition for using the managed properties on the Results and Refiner web parts is that they are listed in the "Selected Properties " in the Results web part. 
Once that is done you should be able to see the values in the debug layout
